src/core/best-practices.js

```diff
@@ -17,20 +16,22 @@ export function run() {
   for (const bp of bps) {
     num++;
     const id = addId(bp, "bp");
-    const li = hyperHTML`<li><a href="${`#${id}`}">Best Practice ${num}</a>: ${
-      bp.textContent
-    }</li>`;
+    const li = hyperHTML`<li><a href="${`#${id}`}">Best Practice ${num}</a>: ${makeSafeCopy(
+      bp
+    )}</li>`;
     ul.appendChild(li);
-    ul.classList.add("practicebox");
-    const title = hyperHTML`<span class="practicetitle">Best Practice ${num}</span>: `;
-    bp.prepend(...title.childNodes);
+    const container = bp.closest("div");
+    if (!container) {
+      bp.classList.add("advisement");
+      continue;
+    }
+    container.classList.add("advisement");
+    const marker = hyperHTML`<a href="${`#${id}`}"
```

Comment:
This is not really a `marker`, it's `contents` of the container that includes a marker right?

Reply:
Yeah, it’s a weird one. It’s as close as I could get it to what the TR design guide recommends.

Reply:
I mean the variable name 😁, the markup includes a marker and the text, so IMO not really appropriate to name it just `marker`...